parser/src/parser.cpp

```diff
@@ -373,6 +373,11 @@ int main(int argc, char* argv[])
     LOG(info) << "The number of changed files exceeds the given incremental "
                  "threshold ratio, full parse will be forced.";
     vm.insert(std::make_pair("force", po::variable_value()));
+
+    cc::util::removeTables(db, SQL_DIR);
+    cc::util::createTables(db, SQL_DIR);
+
+    srcMgr.cacheFiles();
```

Comment:
The `ParserContext ctx` also has a `fileStatus` cache, containing the incremental status of all files.
Probably it is not used further in case of a "clean" parse, but still should be cleared.